aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -30,9 +30,14 @@ module aptos_token::token {
 
     const MAX_COLLECTION_NAME_LENGTH: u64 = 128;
     const MAX_NFT_NAME_LENGTH: u64 = 128;
-    // URI lengths: Mean: 76.97, StdDev: 37.41, 95th%: 157, 99th%: 199 (http://www.supermind.org/blog/740/average-length-of-a-url-part-2)
```

Comment:
why remove this?

Reply:
This is more of explaining why choosing this number instead of specifying what this field means. I don't think this is not the right place for this reasoning